{{cookiecutter.repo_name}}/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import setup, find_packages
 import versioneer
 
-short_description = __doc__.split("\n")
```

Comment:
If the setup.py file is going to have a docstring at all, it should probably describe the setup.py file (and/or how to use it) and not the project. I think it is beyond the scope of the present change, but I would think more appropriate content would be along the lines of
```
Script for setuptools package installation.

Setup file originally generated with [MolSSI CookieCutter](https://github.com/MolSSI/cookiecutter-cms)
at {{cookiecutter.timestamp}} for {{cookiecutter.project_name}}.
After project generation, this script is manually maintained by the project author(s).
Note, though, that this file will be overwritten if ``cookiecutter`` is re-run.
```
I haven't tried this. It might require additional code in one of the hooks scripts: https://cookiecutter.readthedocs.io/en/latest/advanced/injecting_context.html#example-injecting-a-timestamp

But I think it might make more sense for these comments to be regular comments rather than a file docstring.

Reply:
You're right about changing the `short_description` here from the `setup.py` docstring to what is provided: https://docs.python.org/3/distutils/setupscript.html#additional-meta-data. 

As for the docstring of `setup.py` itself, I can't find any rules or conventions on it since every `setup.py` fundamentally does the same thing (I suspect). I like adding the straightforward string you have there though.

Reply:
Reraised as separate issue.